cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -6,6 +6,9 @@
 
 INCLUDE("${CMAKE_CURRENT_LIST_DIR}/utils/ATDMDevEnvUtils.cmake")
 
+# Ensure that LDFLAGS is picked up by the CMake compiler checks
+SET(CMAKE_EXE_LINKER_FLAGS "$ENV{LDFLAGS} ${CMAKE_EXE_LINKER_FLAGS}")
```

Comment:
But it occurred to me that this might actually break configurations with other compilers and systems (I have seen crazy things like this before).  But since this branch has already been merged to 'atdm-nightly' and we have not seen any problems, I guess it is okay.